src/main/java/meteordevelopment/meteorclient/systems/modules/misc/InventoryTweaks.java

```diff
@@ -33,6 +36,8 @@
 import java.util.concurrent.ThreadLocalRandom;
 
 public class InventoryTweaks extends Module {
+    private boolean invOpened = false;
```

Comment:
put this below the settings, like all other fields in modules

Reply:
ok, but then i'll have to remove [line 75](https://github.com/MeteorDevelopment/meteor-client/blob/792682a7e7d19fce874582a4784691fb665f75d2/src/main/java/meteordevelopment/meteorclient/systems/modules/misc/InventoryTweaks.java#L75)